CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contribution Guide
```

Comment:
ignore this file.not from me

Reply:
Why is this file showing up in your PR?

Reply:
don't know why and how it showed up.. locally when i was committing it was never present, but when i created a PR it came as a commit file..
shall i close this PR and raise a new one??